packages/v4/src/content/design-guidelines/components/form-control/form-control.md

```diff
@@ -3,5 +3,54 @@ id: Form control
 section: components
 ---
 
-We’re working on some guidelines for this component.
-If you have any questions in the meantime, feel free to [get in touch with us.](/get-in-touch)
+**Form controls** refer to the different visual states that data input components can take to help guide users in completing forms.
+
+Form control can be applied to various input components, including text boxes, dropdowns/select lists, and text areas.
+
+## Control types
+While form control styles differ slightly depending on the component, they all use some combination of underlines, color, and icons to give users **feedback** on their inputs and selections. For example, if a user types an invalid string into a text box, the box changes state to indicate the error.
+
+<img src="./img/formControlTypes.png" alt="Form control types" />
+
+1. **Default component:** State of the input component before any form control is applied.
+2. **Status area:** Icon and underline color indicate the status of the input or selection.
+3. **Grey-out controls:** Actionless form control can be either disabled or read only. The only difference visually is the underline for read only input.
+4. **Form fields with icon:** Custom icons can be included in input fields.
+
+### Status/state controls
+Status controls are the most prominent form control. Error, warning, and success variations should be used in real time to give users feedback on their inputs or selections.
+
+#### Error controls
+Use error controls when a user inputs incorrect information or selects an invalid option from a dropdown menu. Error indicators are often accompanied by helper text to further specify the user error.
```

Comment:
Now thinking it would be nice to link to the form guidelines about error text. So maybe saying "Error indicators are often accompanied by helper text to further specify the user error. Read more on error messaging in the [forms guidelines](/components/form/design-guidelines#errors-and-validation)